simple_cbor.py

```
@@ -152,431 +320,335 @@ def _decode_simple(self, additional_info: int) -> Any:
+    # PYTHON INTERFACE - Dictionary/List-like access
+    # ========================================================================
+    
+    def __repr__(self) -> str:
+        """String representation."""
+        return f"CBOR({self.data!r})"
+    
+    def __str__(self) -> str:
+        """String representation as diagnostic dump."""
+        try:
+            return self.diag()
+        except:
```

Ruff (0.15.0)

[error] Do not use bare `except`

(E722)